node/hubController.js

```diff
@@ -153,6 +153,73 @@ class HubController {
     }
 
     /**
+     * Subscribe for notifications on all resources composing a platform.  Notifications will be posted to to
+     * the callbackURL.
+     */
+    subscribePlatform(hubId, authInfo, opent2tBlob, callbackUrl) {
```

Comment:
can you add tests? I believe these tests are end to end using a device so that should be ok for subscription as well